sys/arm64/arm64/exception.S

```diff
@@ -129,20 +129,26 @@ __FBSDID("$FreeBSD$");
 	ldp	CAP(12), CAP(13), [sp, #(TF_X + 12 * CAP_WIDTH)]
 	ldp	CAP(14), CAP(15), [sp, #(TF_X + 14 * CAP_WIDTH)]
 	ldp	CAP(16), CAP(17), [sp, #(TF_X + 16 * CAP_WIDTH)]
+#if !__has_feature(capabilities) || defined(__CHERI_PURE_CAPABILITY__)
```

Comment:
If we see more instances of this ifdef we might want to add a new
`__has_feature(cheri_hybrid)` or `__CHERI_HYBRID_CAPABILITY__`

Reply:
There are definitely some other places that check for hybrid in the kernel, but I don't know that it's widespread enough to warrant a new macro:

```
sys/arm64/include/asm.h:#if __has_feature(capabilities) && !defined(__CHERI_PURE_CAPABILITY__)
sys/dev/pci/pci_user.c:#if __has_feature(capabilities) && !defined(__CHERI_PURE_CAPABILITY__)
sys/kern/vfs_vnops.c:#if __has_feature(capabilities) && !defined(__CHERI_PURE_CAPABILITY__)
sys/mips/mips/trap.c:#if __has_feature(capabilities) && !defined(__CHERI_PURE_CAPABILITY__)
sys/riscv/riscv/copyinout.S:#if __has_feature(capabilities) && !defined(__CHERI_PURE_CAPABILITY__)
sys/riscv/riscv/copyinout.S:#if __has_feature(capabilities) && !defined(__CHERI_PURE_CAPABILITY__)
sys/riscv/riscv/copyinout.S:#if __has_feature(capabilities) && !defined(__CHERI_PURE_CAPABILITY__)
sys/sys/systm.h:#if __has_feature(capabilities) && !defined(__CHERI_PURE_CAPABILITY__)
sys/sys/systm.h:#if __has_feature(capabilities) && !defined(__CHERI_PURE_CAPABILITY__)
sys/sys/systm.h:#if __has_feature(capabilities) && !defined(__CHERI_PURE_CAPABILITY__)
```